src/popup/components/AccountCard.vue

```diff
@@ -54,10 +55,18 @@ export default defineComponent({
     selected: Boolean,
   },
   setup(props, { root }) {
+    const { loadTokenBalances } = useFungibleTokens({
+      store: root.$store,
+      accountAddress: props.account.address,
+    });
     const { balance } = useBalances({ store: root.$store });
 
     const numericBalance = computed<number>(() => balance.value.toNumber());
 
+    onMounted(() => {
+      loadTokenBalances();
```

Comment:
This will result in `accounts.length` number of loading started simultaneiously on each `/account` page openning.